src/main/java/com/hubspot/jinjava/doc/JinjavaDocFactory.java

```diff
@@ -4,6 +4,7 @@
 import com.hubspot.jinjava.doc.annotations.JinjavaMetaValue;
 import com.hubspot.jinjava.doc.annotations.JinjavaParam;
 import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.doc.annotations.JinjavaVSCodeSnippet;
```

Comment:
I'm not sure if this should be VSCode specific. This syntax is technically borrowed from [Textmate](https://macromates.com/manual/en/snippets) and other editors use a similar format so I _think_ this could be used across other editors pretty easily